.github/workflows/aot-check.yml

```diff
@@ -23,7 +23,7 @@ jobs:
         fetch-depth: 1
 
     - name: Setup .NET 9.0.x
-      uses: actions/setup-dotnet@v4.1.0
+      uses: actions/setup-dotnet@v4
```

Comment:
@sebastienros Does targeting only a major version here always pulls the latest version of the action? (While targeting major.minor.patch will pin to that specific one?) Or is it that just the major action versions where updated with the fix?

Reply:
Correct, you could have done it on purpose because you want your builds to be super deterministic, but in this case the expected behavior won't change: install dotnet.

Reply:
https://github.com/actions/setup-dotnet/issues/570

Reply:
And this one raising your exact concern https://github.com/dotnet/core/issues/9671#issuecomment-2569473755

Reply:
Thanks @sebastienros